components/RegistrationForm/RegistrationForm.css

```diff
@@ -11,3 +11,8 @@
 .topMargin {
   margin-top: 1rem;
 }
+
+p {
```

Comment:
We can't make style changes to an element because unfortunately this will be applied globally since style code-splitting doesn't work right now.

Look up `CSS Modules` and see how you can create a CSS class and reference it from within `RegistrationForm.js` - it's done all over our app too.

The easiest move would probably be to just do:
```
.RegistrationForm > p {
/* stuff here */
}
```

so that it's scoped to just this component.